### PR TITLE
Fix eslint

### PR DESCRIPTION
**Pull request by su-chang**
Fix eslint issues.

### CODE REVIEW

File: src/manager/padpro-manager.ts [hunk 1]
```diff
@@ -418,7 +418,7 @@ export class PadproManager extends PadproGrpc {
     await CacheManager.init(this.options.token, userId)
     this.cacheManager = CacheManager.Instance
 
-    void this.initData()
+    await this.initData()
```
Comment (windmemory) [inline src/manager/padpro-manager.ts]: why change to await?

File: src/pure-function-helpers/message-raw-payload-parser.ts [hunk 2]
```diff
@@ -14,22 +14,11 @@ import {
...
-import {
-  messageType,
-}                         from './message-type'
 
 import {
-  log,
-}                         from '../config'
+  MessagePayload,
+  MessageType,
+}                         from 'wechaty-puppet'
```
Comment (windmemory) [inline src/pure-function-helpers/message-raw-payload-parser.ts]: Let's put this at the top, and one empty line after this.

File: src/utils/cdn-utils.ts [hunk 3]
```diff
@@ -20,47 +20,47 @@ export const packDownloadRequest = (request: CDNDownloadDataRequest): Buffer =>
 export const unpackDownloadRequest = (buf: Buffer): CDNDownloadDataRequest => {
```
Comment (windmemory) [inline src/utils/cdn-utils.ts]: We need to keep the order of the parameters, this file needs a revert.

Comment (su-chang) [reply src/utils/cdn-utils.ts]: u mean the file `cdn-utils.spec.ts`?

File: src/manager/padpro-manager.ts (hunk 1 above)
Comment (su-chang) [reply src/manager/padpro-manager.ts]: Promises must be handled appropriately, `void ` can't run eslint, `await` did.

**Comment by su-chang** [comment]
1. `npm run test` all results ok.
2. test upload file(image, video, doc) in wechaty Local_Env.

@windmemory confirm pls. Thx.

File: src/manager/padpro-manager.ts [hunk 4]
```diff
@@ -987,7 +986,7 @@ export class PadproManager extends PadproGrpc {
     for (const memberPayload of memberList) {
       const contactId  = memberPayload.Username
       const contact = convertMemberToContact(memberPayload)
-      if (await !this.cacheManager.hasContact(contactId)) {
+      if (!await this.cacheManager.hasContact(contactId)) {
```
Comment (huan) [inline src/manager/padpro-manager.ts]: aha!

File: src/manager/padpro-manager.ts [hunk 5]
```diff
@@ -1038,7 +1037,7 @@ export class PadproManager extends PadproGrpc {
         throw new Error('no cache')
       }
 
-      if (this.cacheManager.hasContact(contactId)) {
+      if (await this.cacheManager.hasContact(contactId)) {
```
Comment (huan) [inline src/manager/padpro-manager.ts]: aha!

File: src/manager/padpro-manager.ts [hunk 6]
```diff
@@ -1071,7 +1070,7 @@ export class PadproManager extends PadproGrpc {
         throw new Error('no cache')
       }
 
-      if (this.cacheManager.hasRoom(id)) {
+      if (await this.cacheManager.hasRoom(id)) {
```
Comment (huan) [inline src/manager/padpro-manager.ts]: and aha!

File: src/pure-function-helpers/app-message-generator.ts [hunk 7]
```diff
@@ -2,125 +2,129 @@ import { UrlLinkPayload } from 'wechaty-puppet'
...
 export const generateLocationMessage = (payload: PadproLocationMessagePayload): string => {
-  return `<location x="${payload.x || ''}"` +
-                  ` y="${payload.y || ''}"` +
-                  ` scale="${payload.scale || ''}"` +
-                  ` label="${payload.label || ''}"` +
-                  ` maptype="${payload.mapType || ''}"` +
-                  ` poiname="${payload.poiName || ''}"` +
-                  ` poiid="${payload.poiId || ''}"` +
-                  ` fromusername="${payload.fromUsername || ''}" />`
+  return `
```
Comment (huan) [inline src/pure-function-helpers/app-message-generator.ts]: This is bad.

use

```ts
[
  'a',
  'b',
].join('')
```

instead.

File: src/manager/padpro-manager.ts (hunk 4 above)
Comment (windmemory) [reply src/manager/padpro-manager.ts]: yep, this is the bug that you caught!

**Pull request merged**